classes/class-wpcom-liveblog-entry-embed-sdks.php

```diff
@@ -39,7 +39,11 @@ public static function enqueue() {
 		}
 
 		foreach ( self::$sdks as $name => $url ) {
-			wp_enqueue_script( $name, esc_url( $url ), array(), WPCOM_Liveblog::VERSION, false );
+			if ( 'reddit' === $name ) {
+				wp_enqueue_script( $name, esc_url( $url ), array(), null, false );
```

Comment:
For readability, is it possible to leave a comment for why we are doing it this way and maybe something like this?

```suggestion
                                 $version = 'reddit' === $name ? null : WPCOM_Liveblog::VERSION;
				wp_enqueue_script( $name, esc_url( $url ), array(), null, false );
```